packages/ui-popover/src/Popover/index.tsx

```diff
@@ -579,7 +595,14 @@ class Popover extends Component<PopoverProps, PopoverState> {
           ...(color === 'primary-inverse' && { borderColor: 'transparent' })
         } as Partial<ViewProps> & { ref: any }
 
-        return <View {...viewProps}>{content}</View>
+        return (
+          <View
+            {...viewProps}
+            themeOverride={{ borderColorPrimary: styles?.borderColor }}
```

Comment:
we shouldn't use `themeOverride` for our own components

Reply:
removed